test/unittests/evm_test.cpp

```diff
@@ -777,35 +777,174 @@ TEST_F(evm, selfdestruct)
 
 TEST_F(evm, selfdestruct_with_balance)
 {
-    auto code = "6000ff";
-    msg.destination.bytes[0] = 1;
+    const auto beneficiary = evmc::address{{0xbe}};
+    const auto code = push({beneficiary.bytes, sizeof(beneficiary)}) + OP_SELFDESTRUCT;
+    msg.destination = evmc_address{{0x5e}};
+
+
+    accounts[msg.destination].set_balance(0);
+
+    rev = EVMC_HOMESTEAD;
+    execute(3, code);
+    EXPECT_GAS_USED(EVMC_SUCCESS, 3);
```

Comment:
Why does this self-destruct cost 3 gas on Homestead, I thought they were free until EIP150 (Tangerine Whistle)?

Reply:
The 3 is from the PUSH preceding it.